### ISSUE TITLE
Voice match: this feature is not available in your language

### ISSUE THREAD

**Issue by Radplay** [PixelExperience/android-issues#60]
## Build date

```
PixelExperience_lavender-11.0-20210419-2113-OFFICIAL
```

## Expected Behavior

```
Voice match should be toggleable
```

## Current Behavior

```
Voice match says it's not available in your language, no matter which language it's set to. It can't be turned on
```

## Possible Solution

```
An update should fix that
```

## Steps to Reproduce

```
1. Open Google Assistant
2. Go to voice match
3. It says it's not available in your language no
```

/device lavender
/version eleven

**Comment by georgethemarmot** [comment]
Same issue on 
/device tissot
/version eleven
PixelExperience_tissot-11.0-20210517-1224-OFFICIAL.zip

**Comment by Hlcpereira** [comment]
Fixed on build PixelExperience_lavender-11.0-20210522-1047-OFFICIAL
